modules/client/examples.md

```diff
@@ -481,8 +482,7 @@ const pluginInitParams1: ITokenVotingPluginInstall = {
     minParticipation: 0.25, // 25%
     supportThreshold: 0.5, // 50%
     minProposerVotingPower: BigInt("5000"), // default 0
-    earlyExecution: true, // default false
-    voteReplacement: false, // default false,
+    votingMode: VotingMode.STANDARD, // default standard
```

Comment:
I'm not sure "standard" is easy enough to understand, as I don't think there is a "standard" way of voting. If the enum 1:1 matches the smart contract one, then we might rather address this at the contract level and not here.

- Would "DEFAULT" be any more logical for an SDK consumer?
- Do we have any example for vote replacement?
- Otherwise, I would maybe leave the two booleans with a default to false, and throwing an error when both are improperly combined

Reply:
Yes this matches 1:1 the contracts, https://github.com/aragon/core/blob/420c6346cb750d58b434d3d98a6178c24c4729fd/packages/contracts/test/test-utils/voting.ts#L12
 No we dont have an example for vote replacement

Reply:
Because it will be implemented later?

Reply:
I added an example with `VotingMode.VOTE_REPLACEMENT`, it is exactly the same 👍